index.ts

```diff
@@ -84,17 +85,20 @@ export async function setupPlugin({ config, global }: BrazeMeta): Promise<void>
                 timeout: 5000,
             })
         } catch (e) {
-            console.error(e)
+            console.error(e, endpoint, options.body, eventUuid)
             throw new RetryError('Fetch failed, retrying.')
```

Comment:
Ah also, additionally, we should not be retrying e.g. 400 errors, something I missed previously!

Essentially we only want to retry on intermittent errors e.g. 5xx or not response.

Reply:
For example, if we have a payload that is too large and get 413 entity too large, we will continuously churn on these.

Reply:
Looks like that is handled further down as well, so I think we just want to remove the `throw` 🤔